Library/Formula/mftrace.rb

```diff
@@ -1,15 +1,18 @@
-require 'formula'
-
 class Mftrace < Formula
-  homepage 'http://lilypond.org/mftrace/'
-  url 'http://lilypond.org/download/sources/mftrace/mftrace-1.2.18.tar.gz'
-  sha1 '8ae5c69a49f3d34e021913b6378056a6d43c2fd2'
+  homepage "http://lilypond.org/mftrace/"
+  url "http://lilypond.org/download/sources/mftrace/mftrace-1.2.18.tar.gz"
+  sha256 "0d31065f1d35919e311d9170bbfcdacc58736e3f783311411ed1277aa09d3261"
 
-  depends_on 'potrace'
-  depends_on 't1utils'
+  depends_on "potrace"
+  depends_on "t1utils"
+  depends_on "fontforge" => [:recommended, :run]
 
   def install
     system "./configure", "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/mftrace", "-v"
```

Comment:
Seems to fail: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/22673/version=yosemite/testReport/junit/brew-test-bot/yosemite/test_mftrace/